src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -129,12 +130,7 @@
         } else {
             if (password!=null) {
                 String authorization = (login + ':' + password);
-                final Charset charset;
-                try {
-                    charset = Charset.forName("UTF-8");
-                } catch (Exception ex) {
-                    throw new IOException("UTF-8 encoding is not supported", ex);
-                }
+                Charset charset = Charsets.UTF_8;
```

Comment:
If more people agree I'll do that then.
Shall I remove the variable declaration and inline it to the method call or just convert to static import?